specification/07-theory_operations.adoc

```diff
@@ -135,7 +135,7 @@ initiated CoVE-IO `COVH` calls that require SPDM requests to be sent to the devi
 DOE mailbox follows the flow described below:
 
 .SPDM Flow With CoVE
-[source,mermaid]
+[mermaid, ,svg]
```

Comment:
Did you mean `mermaid, svg`?

Reply:
Nope, I was following the asciidoctor diagram block syntax: https://docs.asciidoctor.org/diagram-extension/latest/blocks/

Reply:
Unfortunately, that seems to break the markdown rendering, and does not fix the PDF generation neither.